src/linkml_map/inference/inference.py

```diff
@@ -36,8 +36,14 @@ def induce_missing_values(
                 # auto-populate range field
                 if cd.populated_from not in source_schemaview.all_classes():
                     continue
+
+                populated_from_slot = sd.populated_from
+                if "." in populated_from_slot:
+                    # Skip range inference for FK lookups (e.g., "org_id.name")
```

Comment:
seems this could be done using the same logic as you implemented in object_transformer. I'll also comment there